### PR TITLE
fix: [M3-7142] - Update helper text copy in NodeBalancer Create form 'Algorithm' field

### PR DESCRIPTION
**Pull request by coliu-akamai**
## Description 📝
- Updates helper text for Round Robin algorithm and only shows helper text relevant to the selected algorithm

## Changes
- Updated helper text copy for the 'algorithm' field + updated additional helper text copies as specified
- Added unit tests for config panel 
- some eslint fixes whenever I landed on a file that had them

## Target release date 🗓️
n/a

## Preview 📷

| Before  | After   |
| ------- | ------- |
| image | image imageimage |

### Reproduction steps
- Navigate to NodeBalancer create form, note the helper text underneath the 'Algorithm' field

### Verification steps
- Confirm copies for each algorithm field as specified by ticket
- confirm create flow still works as expected
- `yarn test NodeBalancerConfigPanel.test.tsx`

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.test.tsx [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+const node: NodeBalancerConfigNodeFields = {
+  address: '',
+  label: '',
+  mode: 'accept',
+  modifyStatus: 'new',
+  port: 80,
+  weight: 100,
+};
+
+const props: NodeBalancerConfigPanelProps = {
```
Comment (mjac0bs) [inline packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.test.tsx:31]: Wow. 😅

File: packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.test.tsx [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+    const { getByText } = renderWithTheme(
+      <NodeBalancerConfigPanel {...props} nodes={[{ ...node }, node]} />
+    );
+
+    const removeNodeButton = getByText('Remove');
+    fireEvent.click(removeNodeButton);
+    expect(props.removeNode).toHaveBeenCalled();
+  });
+
+  it('deletes the confirguration panel', () => {
```
Comment (mjac0bs) [inline packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.test.tsx:244]:
```suggestion
  it('deletes the configuration panel', () => {
```
nit: typo

File: packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.test.tsx [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    const { getByText } = renderWithTheme(
+      <NodeBalancerConfigPanel {...props} />
+    );
+
+    const deleteConfigButton = getByText('Delete');
+    fireEvent.click(deleteConfigButton);
+    expect(props.onDelete).toHaveBeenCalled();
```
Comment (mjac0bs) [inline packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.test.tsx:251]: If we test the Cancel button too, we'd see a failure! Looks like M3-8508 just came in as a bug report, so we can expand upon this test then.

**Review by mjac0bs — APPROVED** [review]
Our test coverage of NodeBalancers has been lacking for a long time and your additional coverage here is awesome to have! 

Confirmed the config panel Algorithm field looks good now and helper text across the form is consistent. Thanks for tackling this one. The original typo had been around for 5+ years - I remember checking when I made the ticket.

**Review by dwiley-akamai — APPROVED** [review]
`NodeBalancerConfigPanel.test.tsx` passes locally and in CI ✅
Confirmed copy updates when a different Algorithm is selected ✅
No regressions observed when creating a NodeBalancer ✅
Code review ✅

File: packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.test.tsx (hunk 3 above)
Comment (coliu-akamai) [reply packages/manager/src/features/NodeBalancers/NodeBalancerConfigPanel.test.tsx:251]: Made a note on that ticket to expand this test 😄

**Pull request merged**